src/main/java/bio/terra/common/stairway/StairwayJdbcConfiguration.java

```diff
@@ -0,0 +1,11 @@
+package bio.terra.common.stairway;
+
+import bio.terra.common.db.JdbcConfiguration;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "buffer.stairway.db")
```

Comment:
prefix should be the `common.stairway.db`

Reply:
`terra.common.stairway.db` if we want to match the tracing package.

Reply:
`terra.common.stairway.db`?